frontend/src/metabase/admin/settings/components/SettingsSetting.jsx

```diff
@@ -33,62 +35,85 @@ const SETTING_WIDGET_MAP = {
   hidden: () => null,
 };
 
-export default class SettingsSetting extends Component {
-  static propTypes = {
-    setting: PropTypes.object.isRequired,
-    settingValues: PropTypes.object,
-    onChange: PropTypes.func.isRequired,
-    onChangeSetting: PropTypes.func,
-    autoFocus: PropTypes.bool,
-    disabled: PropTypes.bool,
-  };
+const SettingSetting = props => {
+  const { hash } = useLocation();
+  const [fancyStyle, setFancyStyle] = useState({});
+  const thisRef = useRef();
+
+  const { setting, settingValues, errorMessage } = props;
+
+  useEffect(() => {
+    if (hash === `#${setting.key}`) {
+      setTimeout(() => {
+        thisRef.current.scrollIntoView({
+          behavior: "smooth",
+          block: "start",
+        });
```

Comment:
could this be useful?

https://v6.mantine.dev/hooks/use-scroll-into-view/

Reply:
I couldn't actually get this to work for me haha